install/kubernetes/helm/istio/charts/gateways/templates/deployment.yaml

```diff
@@ -108,6 +108,10 @@ spec:
               fieldRef:
                 fieldPath: metadata.name
           volumeMounts:
+          {{- if $.Values.global.mtls.enabled }}
+          - name: sdsudspath
```

Comment:
I think we need this no matter the flag is enabled or not. Think about turn on mTLS on the fly by auth policy. If we add it here always, that would reduce the mTLS enablement effort.

Reply:
sounds reasonable, updated.